api/reference/vms.adoc

```diff
@@ -163,6 +163,7 @@ GET /api/vms/:id?expand=accounts,software
   "action" : "request_retire"
 }
 ----
+Note that there are two retirement endpoints. Retirement as a request (RaaR) was added in our release of Hammer and replaces retire_now as the default retirement method. The retire_now method is kept for backwards compatibility but can be used only with a pre-Hammer Automate state machine.
```

Comment:
I wouldn't add this to the doc since we only want folks to use request_retire. Just the change to the table above to reflect "request_retire". Thanks.